cvat-ui/src/components/annotation-page/top-bar/left-group.tsx

```diff
@@ -26,11 +27,32 @@ interface Props {
     undoShortcut: string;
     redoShortcut: string;
     drawShortcut: string;
+    blockShortcut: string;
+    toolsBlockerState: ToolsBlockerState;
     activeControl: ActiveControl;
     onSaveAnnotation(): void;
     onUndoClick(): void;
     onRedoClick(): void;
     onFinishDraw(): void;
+    onSwitchBlockMode(): void;
+}
+
+interface StateToProps {
+    toolsBlockerState: ToolsBlockerState;
+}
+
+function mapStateToProps(state: CombinedState): StateToProps {
+    const {
+        settings: {
+            workspace: {
+                toolsBlockerState,
+            },
+        },
+    } = state;
+
+    return {
+        toolsBlockerState,
+    };
```

Comment:
That must be refactored for using hooks, but for now let's get ``toolsBlockerState`` from TopBarContainer and pass it here via props TopBarContainer > TopBarComponent > LeftGroup. Just to keep code consistent

Reply:
BTW ``onSwitchBlockMode`` recieved exactly this way.

Reply:
Done